components/TextFields/src/MDCTextInputControllerFloatingPlaceholder.h

```diff
@@ -0,0 +1,74 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCTextInputController.h"
+
+/** Controllers that have the ability to move the placeholder to a title position. */
+@protocol MDCTextInputControllerFloatingPlaceholder <MDCTextInputController>
+
+/**
+ The color applied to the placeholder when floating. However, when in error state, it will be
+ colored with the error color and when in active state, it will be colored with the active color.
+
+ Only relevent when floatingEnabled = true.
```

Comment:
super-nit, since this is Obj-C, I feel like the default docs should use ObjC values. Either:

```
Only relevant when floatingEnabled = YES.
```
or
```
Only relevant when floatingEnabled is true.
```

Reply:
Done